src/main/frontend/src/App.js

```diff
@@ -1,23 +1,20 @@
 import logo from './logo.svg';
 import './App.css';
+import Button from 'react-bootstrap/Button'
+import 'bootstrap/dist/css/bootstrap.min.css'
+import Navigation from './components/Navigation';
+import Footer from './components/Footer';
+import HeaderAboveNav from './components/HeaderAboveNav';
+import Home from './pages/Home';
+
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <HeaderAboveNav />
+      <Navigation />
+      <Home />
+      <Footer />
```

Comment:
It would be better to create wrapper component called `PageLayout` that would display headers and footer.
Component render method would eb something like:
```suggestion
render( 
      <HeaderAboveNav />
      <Navigation />
      { children }
      <Footer />
      )
```
and then here can be called as 
```suggestion
      <PageLayout>
          <Home />
      </PageLayout>
```